src/web/components/Core/Collapsible.tsx

```diff
@@ -0,0 +1,31 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import * as RadixCollapsible from '@radix-ui/react-collapsible';
+import { ReactNode } from 'react';
+
+import './Collapsible.scss';
+
+export type CollapsibleProps = {
+  title: string;
+  content: ReactNode;
+  defaultOpen: boolean;
+};
+
+export function Collapsible({ title, content, defaultOpen }: CollapsibleProps) {
```

Comment:
I've left this as uncontrolled for now, but we can easily make it controlled later per the example here: https://www.radix-ui.com/docs/primitives/components/collapsible